doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1809,6 +2153,14 @@ Run-time optimization tactic
 .. tacn:: optimize_heap
    :name: optimize_heap
 
-   This tactic behaves like :n:`idtac`, except that running it compacts the
+   This tactic behaves like :tacn:`idtac`, except that running it compacts the
    heap in the OCaml run-time system. It is analogous to the Vernacular
    command :cmd:`Optimize Heap`.
+
+.. tacn:: info @ltac_expr
+
+   :tacn:`info` is a :token:`binder_tactic`.
+
+.. tacn:: infoH @ltac_expr3
+
+   :tacn:`infoH` is an :token:`l3_tactic`.
```

Comment:
Since 8.5, `info` says:

```
Warning:
The general "info" tactic is currently not working. There is an "Info" command to replace it.
Some specific verbose tactics may also exist, such as info_eauto. [deprecated-info-tactical,deprecated]
```

and `infoH`:

```
<infoH></infoH>
```

Sounds like a good time to remove them. If you want to do this, please open a separate PR. We could backport it to v8.12 if it is merged before releasing the beta.